data/built-ins.json

```diff
@@ -666,34 +673,23 @@
     "ios": 10,
     "opera": 34
   },
-  "es7.object.values": {
-    "chrome": 54,
-    "edge": 14,
-    "firefox": 47,
-    "node": 7,
-    "opera": 41
-  },
-  "es7.object.entries": {
-    "chrome": 54,
-    "edge": 14,
-    "firefox": 47,
-    "node": 7,
-    "opera": 41
-  },
-  "es7.object.get-own-property-descriptors": {
-    "chrome": 54,
-    "firefox": 50,
-    "node": 7,
-    "opera": 41
-  },
+  "es7.object.values": {},
+  "es7.object.entries": {},
+  "es7.object.get-own-property-descriptors": {},
```

Comment:
because of this https://github.com/kangax/compat-table/commit/5d08a40acbd5744d61e1bb841928a11380b87d6f